tests/test_user.py

```diff
@@ -458,3 +461,504 @@ def test_user_settings_edit_dev_app(self, driver, session, fake):
         finally:
             # Lastly use the api to delete the dev app as cleanup
             osf_api.delete_user_developer_app(session, app_id=app_id)
+
+
+@markers.dont_run_on_prod
+@pytest.mark.usefixtures('must_be_logged_in')
+class TestUserPersonalAccessTokens:
+    def create_scope_dictionary(self, driver):
+        """Helper function used to create a dictionary of all of the scope checkboxes on
+        the Edit Personal Access Token Page.
+        """
+        all_scope_rows = driver.find_elements_by_css_selector(
+            '._Checkboxes_qxt8ij > div'
+        )
+        scope_dictionary = {}
+        for row in all_scope_rows:
+            scope = row.get_attribute('data-test-scope')
+            checkbox_element = row.find_element_by_css_selector('input')
+            scope_dictionary[scope] = checkbox_element
+        return scope_dictionary
+
+    def verify_scope_checkboxes(self, driver, page, scope_list, checked=True):
+        """Helper function used to verify scope checkboxes on the Edit Personal Access
+        Token Page. For each scope in the scope_list parameter, get the corresponding
+        checkbox element from the scope dictionary. Then verify whether the checkbox
+        should be selected or not based on the checked parameter.
+        """
+        scope_dictionary = self.create_scope_dictionary(driver)
+        for scope in scope_list:
+            checkbox = scope_dictionary[scope]
+            page.scroll_into_view(checkbox)
+            if checked:
+                assert checkbox.is_selected()
+            else:
+                assert not checkbox.is_selected()
```

Comment:
Can simplify the prior four rows to: `assert checkbox.is_selected() == checked`

Reply:
D'oh!  I'll fix that. Thanks!

Reply:
Fixed.